### ISSUE TITLE
File closing: Callback is not a function in Node 10 

### ISSUE THREAD

**Issue by megastef** [mingqi/tailF#12]
Hi, i get this error in Node 10 - do you have an idea how to fix it? 

```
TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at makeCallback (fs.js:136:11)
    at Object.close (fs.js:394:20)
    at /usr/local/lib/node_modules/@sematext/logagent/node_modules/tail-forever/tail-forever.js:107:18
    at FSReqWrap.oncomplete (fs.js:154:5)
```

I think the line number does not match (using 0.3.15): https://github.com/mingqi/tailF/blob/master/tail-forever.js#L107

**Comment by BridgeAR** [comment]
This is fixed on master but no new version has ever been released.

**Comment by otisg** [comment]
Good to know, @BridgeAR, thanks!  
@mingqi any chance you could make a new release?

**Comment by megastef** [comment]
I published "tail-forever-2" on npm to have a working package in npm. 
There was no  async fs function call without callback. The new package includes also  a fix for deprecated Buffer constructor. PR https://github.com/mingqi/tailF/pull/13 

I would be glad to remove the temporary npm package, in case @mingqi  is able to update the original package.

**Comment by mingqi** [comment]
> I published "tail-forever-2" on npm to have a working package in npm.
> There was no async fs function call without callback. The new package includes also a fix for deprecated Buffer constructor. PR #13
> 
> I would be glad to remove the temporary npm package, in case @mingqi is able to update the original package.

Hi @megastef , I can add you into the NPM pacakge "tail-forvever" as a maintainer in order you can publish pacakge yourself.  Please let me know your username in NPM(npmjs.com).  I guess it's **sematext**, author of "tail-forever-2".  I just have invited sematext as the maintainer of package "tail-forever"

**Comment by megastef** [comment]
@mingqi thank you. My personal account is  https://www.npmjs.com/~megastef however sematext https://www.npmjs.com/~sematext  account is better - so even one of my colleagues could give a helping hand. I will remove tail-forever-2 after updating dependencies ... and new Logagent release.

**Comment by megastef** [comment]
@mingqi could you add me as GH maintainer? So i could increase version number in package.json for publishing on NPM. I use the release-it tool.

**Comment by megastef** [comment]
I removed tail-forever-2 from npm and released tail-forever:0.3.16
